include/nil/actor/zk/snark/systems/plonk/placeholder/proof.hpp

```diff
@@ -32,34 +32,42 @@ namespace nil {
     namespace actor {
         namespace zk {
             namespace snark {
+                constexpr std::size_t FIXED_VALUES_BATCH = 0;
```

Comment:
Looks like good work for enum :)

Reply:
Let's change this later, these constants are copied twice in this code, and used all over the place, no time to change this now.